Tasks/Common/nuget-task-common/NuGetToolRunner.ts

```diff
@@ -139,7 +141,11 @@ function locateTool(tool: string, opts?: LocateOptions) {
 
 export function locateNuGetExe(userNuGetExePath: string): string {
     if (userNuGetExePath) {
-        tl.debug(`using user-supplied NuGet path ${userNuGetExePath}`)
+        if (os.platform() === "win32") {
```

Comment:
yep, this is literally just to resolve that one break and even then only on windows.